hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieMergedLogRecordScanner.java

```diff
@@ -110,7 +110,7 @@ protected void processNextRecord(HoodieRecord<? extends HoodieRecordPayload> hoo
     if (records.containsKey(key)) {
       // Merge and store the merged record. The HoodieRecordPayload implementation is free to decide what should be
       // done when a delete (empty payload) is encountered before or after an insert/update.
-      HoodieRecordPayload combinedValue = records.get(key).getData().preCombine(hoodieRecord.getData());
+      HoodieRecordPayload combinedValue = hoodieRecord.getData().preCombine(records.get(key).getData());
```

Comment:
I have changed the order of precombine here. HoodieAvroPayload has a consistent behavior across precombine and combineAndGet APIs where it favors this instance over another. Inorder to keep the same semantics, the ordering needs to be changed here.

Reply:
makes sense

Reply:
Guess there is no correct answer to if `this` instance make sense or `that` right ? So as long as we keep this consistent it should be fine ? For custom payloads, there should be no assumption between this and that ?

Reply:
Right.